app/controllers/users/confirmations_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Users
+  # Overwriting Devise ConfirmationsController
+  class ConfirmationsController < Devise::ConfirmationsController
+    def after_confirmation_path_for(_resource_name, resource)
```

Comment:
this way, when a user is confimed through the link, he automatically is logged in,